### DISCUSSION TITLE
Is is possbile to set the min-height of Q-Select

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#16774] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I reproduce this issue with below link, could you please have a look?
https://codepen.io/Stanley-549393092/pen/YzgzoMx
The background is: it has to embed q-input and q-select in the table.
The q-input is customized with css class zinput and it works well.
I tried to customize q-select with css class zselect but it does not work.
Although it adds min-height: 28px, but somehow it will be overridden.

Regards

**Comment by pdanpdan** [comment, marked as answer]
check the CSS:
https://codepen.io/pdanpdan/pen/KKENLQQ

**Reply by 549393092** [reply]
@pdanpdan It's awesome, thanks!
